evaluation/examples/linux-cpp/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu
```

Comment:
OK, I just thought it would be safer to match the host kernel version. You're right that it might be whatever compatible with our kernel version. (So something too bleeding edge might not be.)

(And I was wandering what is a Docker best practice, but I don't really know.)